internal/stdin/stdin.go

```diff
@@ -30,7 +30,7 @@ func Read() (string, error) {
 		}
 	}
 
-	return strings.TrimSuffix(b.String(), "\n"), nil
+	return strings.TrimSpace(b.String()), nil
```

Comment:
So you are also trimming on the left.

Please confirm if it intended and OK

Reply:
Thought about that, I'm not sure why we were trimming the eof \n... in any case, it's probably fine 🤔